File: pinyougou-interface/src/main/java/cn/itcast/core/service/SeckillOrderService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.seckill.SeckillOrder;
import entity.PageResult;

import java.util.List;

public interface SeckillOrderService {
    List<SeckillOrder> findAll(String name);

    PageResult search( Integer page, Integer rows, SeckillOrder seckillOrder);
}

File: pinyougou-web-shop/src/main/webapp/js/service/seckillOrderService.js
// 定义服务层:
app.service("seckillOrderService",function($http){
	this.findAll = function(){
		return $http.get("../seckillOrder/findAll.do");
	}

	this.search = function(page,rows,searchEntity){
		return $http.post("../seckillOrder/search.do?page="+page+"&rows="+rows,searchEntity);
	}
	

});